code/game/turfs/open/floor.dm

```diff
@@ -44,6 +46,8 @@
 	if(is_station_level(z))
 		GLOB.station_turfs += src
 
+/turf/open/floor/proc/force_plating(forced_plating)
+	make_plating(forced_plating)
```

Comment:
because other floor tiles can be made that will require it, and i dont think it good practice to override it here every single time you add a new floor that normally doesnt want to break, just because this only applies to one tile now, doesnt mean it always will, and i was specifically told by another maintainer to use a proc